Sources/RemoteMessaging/Model/RemoteConfigModel.swift

```diff
@@ -20,5 +20,15 @@ import Foundation
 
 public struct RemoteConfigModel {
     let messages: [RemoteMessageModel]
-    let rules: [Int: [MatchingAttribute]]
+    let rules: [RemoteConfigRule]
```

Comment:
This is technically less efficient than the old implementation, but a bit simpler. We have so few rules at one time that the difference here is negligible, but let me know if you feel otherwise.